packages/editor-ui/src/components/WorkflowActivator.vue

```diff
@@ -74,59 +81,47 @@ export default mixins(
 					}
 					return '#13ce66';
 				},
+				isCurrentWorkflow(): boolean {
+					return this.$store.getters['workflowId'] === this.workflowId;
+				},
+				disabled(): boolean {
+					const isNewWorkflow = !this.workflowId;
+					if (isNewWorkflow || this.isCurrentWorkflow) {
+						return !this.workflowActive && !this.containsTrigger;
```

Comment:
only disable for activators within workflow page.. not the ones in workflows list